rules/windows/process_creation/process_creation_tool_nircmd.yml

```diff
@@ -16,24 +17,20 @@ logsource:
     product: windows
 detection:
     selection:
-        Image|endswith: '\nircmd.exe'
-    selection_params1:
-        CommandLine|contains|all:
-            - ' execmd '
-            - ' attrib ' 
-    selection_params2:
-        CommandLine|contains|all:
-            - ' execmd '
-            - ' copy ' 
-    selection_params3:
-        CommandLine|contains|all:
-            - ' execmd '
-            - ' del '
-            - ' /Q '
+        Image|endswith:
+            - '\nircmd.exe'
+            - '\nircmdc.exe'
+    selection_params:
+        CommandLine|contains:
```

Comment:
this is not specific enough - it would trigger on every commandline that contains an 'exec'

Reply:
Yes, I thought as much. The "exec" keyword is kind of generic. Thanks

Reply:
I've also added spaces before and after the keywords.